appveyor.yml

```diff
@@ -1,3 +1,24 @@
+# The config expects the following environment variables to be set:
+#  - "GN_CONFIG" Build type. One of {'debug', 'testing', 'release'}.
+#  - "GN_EXTRA_ARGS" Additional gn arguments for a build config,
+#      e.g. 'target_cpu="x86"' to build for a 32bit platform.
+#      https://gn.googlesource.com/gn/+/master/docs/reference.md#target_cpu
+#  - "ELECTRON_RELEASE" Define it to upload binaries on success.
+#  - "NPM_CONFIG_ARCH" E.g. 'x86'. Is used to build native Node.js modules.
+#  - "TARGET_ARCH" Choose from {'ia32', 'x64', 'arm', 'arm64', 'mips64el'}.
+#      Is used in some publishing scripts, but does NOT affect the Electron binary.
+#  - "UPLOAD_TO_S3" Define it to upload a release to the S3 bucket.
+#      If it's not defined the release is uploaded to the Github Releases.
+#      The value is only checked if "ELECTRON_RELEASE" is defined.
```

Comment:
if `ELECTRON_RELEASE` causes binaries to be uploaded, what does this do differently?

Reply:
`ELECTRON_RELEASE` means that binaries should _be_ uploaded, `UPLOAD_TO_S3` controls _where_ exactly.

Reply:
oh, is `UPLOAD_TO_S3` supposed to be a bucket name then? That's totally not clear from the comment.

Reply:
No, it's not a bucket =) It's just a flag.
@jkleinsc can probably explain it better.

Reply:
@nornagon `ELECTRON_RELEASE` tells CI to (a) build for release and (b) upload those release files

By default those uploads go to Github Release, if you define `UPLOAD_TO_S3` it will instead upload them to S3